src/components/DisplayInput.js

```diff
@@ -0,0 +1,138 @@
+const React = require('react');
+const Radium = require('radium');
+
+const StyleConstants = require('../constants/Style');
+
+const DisplayInput = React.createClass({
+  propTypes: {
+    hint: React.PropTypes.string,
+    label: React.PropTypes.string,
+    placeholder: React.PropTypes.string,
+    primaryColor: React.PropTypes.string,
+    showHint: React.PropTypes.bool,
+    status: React.PropTypes.shape({
+      type: React.PropTypes.string,
+      message: React.PropTypes.string
+    }),
+    valid: React.PropTypes.bool
+  },
+
+  getDefaultProps () {
+    return {
+      primaryColor: StyleConstants.Colors.PRIMARY,
+      valid: true
+    };
+  },
+
+  render () {
+    const styles = this.styles();
+
+    return (
+      <div style={styles.wrapper}>
+        {this.props.label ?
+          <div key='label' style={styles.label}>
```

Comment:
Without it I get a `Radium requires each element with interactive styles to have a unique key` error.